src/containers/About/styles.module.css

```diff
@@ -0,0 +1,22 @@
+.container{
```

Comment:
Please leave a space between the class names and the opening symbol `{`.
Also, make sure the last line is empty.